website/pages/docs/concepts/seal.mdx

```diff
@@ -89,75 +89,159 @@ Recovery Keys and are used for these operations.
 For a list of examples and supported providers, please see the
 [seal documentation](/docs/configuration/seal).
 
+## Recovery Key Rekeying
+
+During KMS Seal initialization process, a set of Shamir keys called Recovery Keys are
+generated which are used for operations that still require a quorum of users.
+
+Recovery Keys can be rekeyed to change the number of shares or thresholds. When using the
+Vault CLI, this is performed by using the `-target=recovery` flag to `vault operator rekey`.
+
 ## Seal Migration
 
-The seal can be migrated from Shamir keys to Auto Unseal, Auto Unseal to Shamir
-keys and Auto Unseal to another Auto Unseal.
+The seal can be migrated from Shamir Seal to KMS Seal, KMS Seal to Shamir Seal
+and KMS Seal to another KMS Seal.
+
+~> **NOTE**: The seal migration process cannot be performed without downtime.
+Due to the intricate technical underpinnings of the seal feature, it is at this
+point not possible to perform seal migration without bringing the whole cluster
+down. We understand that it can be painful for many users to bring the full
+cluster down. But, we believe that switching seals is a very rare event and
+hence hope for the downtime to be considered as an acceptable trade-off.
 
 ~> **NOTE**: The migration operation will require both seals to be available
-during the migration. For example, a migration from a cloud KMS seal to
-Shamir will require that the cloud KMS be accessible during the migration.
+during the migration. For example, a migration from a KMS seal to Shamir will
+require that the KMS key be accessible during the migration.
+
+### Migration From Shamir Seal to KMS Seal
+
+#### When **not** using Integrated Storage
 
-### Migration From Shamir to Auto Unseal
+1. Take your server cluster offline.
 
-To migrate from Shamir keys to Auto Unseal, take your server cluster offline and update
-the [seal configuration](/docs/configuration/seal) with the appropriate seal
-configuration. Bring your server back up and leave the rest of the nodes offline if
-using multi-server mode, then run the unseal process with the `-migrate` flag and bring
-the rest of the cluster online.
+2. Add the desired KMS [seal block](/docs/configuration/seal) to the
+configuration file of one of the nodes.
 
-All unseal commands must specify the `-migrate` flag. Once the required threshold of
-unseal keys are entered, unseal keys will be migrated to recovery keys.
+3. Bring this one node back up and run the unseal command by supplying the
+`-migrate` flag for each unseal key. This node will be the active node and
+completes the seal migration process. The Shamir unseal keys supplied here will
+be migrated as the recovery keys for the KMS seal.
 
-```
-$ vault operator unseal -migrate
-```
+4. Standby nodes can join the active node's cluster using the new seal blocks.
```

Comment:
I don't understand "using the new seal blocks" - what needs to happen on the standby nodes? Or can you just re-start them and they will join because of the config block on the original node?